elements/_wellcome.scss

```diff
@@ -0,0 +1,55 @@
+.background_wellcome {
+    position: absolute;
+    background-position: bottom;
+    bottom: 0;
+    width: 100vw;
+}
+
+.section_wellcome {
+    height: calc(100vh - 70px);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+
+    @include mobile {
+        height: calc(100vh - 70px);
+    }
+
+    .headline_wellcome{
+        text-align: center;
+        font-stretch: normal;
+        font-size: 49px;
+        line-height: 57px;
+        font-family: $font-raleway;
+        letter-spacing: 0px;
+        color:  $color-black;
+        margin-top: 130px;
+        background-position: bottom;
+    
+        @include mobile {
+            margin-top: 100px;
+            font-size: 35px;
```

Comment:
dobrą praktyką jest ustawianie parzystych rozmiarów czcionek. wtedy łatwiej się to pozycjonuje i jak masz kontener 60px i dasz wyśrodkowanie w pionie to będzie równo.